packages/web/src/components/data-entry/TextAreaV2.tsx

```diff
@@ -19,6 +19,11 @@ const sizeToVerticalPadding: Record<TextAreaSize, number> = {
   [TextAreaSize.SMALL]: 12
 }
 
+const getMaxHeight = (maxVisibleRows: number | undefined, size: TextAreaSize) =>
```

Comment:
feels weird to have a first arg number | undefined, but just a nit. i would do

`getMaxHeight = ({ size, maxVisibleRows} : {size: TextAreaSize, maxVisibleRows?: number})`